client/src/components/ui/ImageCard.style.tsx

```diff
@@ -2,7 +2,7 @@ import styled from "styled-components";
 
 interface SizeOfImageCard {
   alt: string;
-  src: string;
+  src: string | object;
```

Comment:
Why object?

Reply:
Because in db model "photos" is an array of objects and I need to get only one item of this object to display as a main photo. When I use string type I get a source of this image, but as string, not image. 
![image](https://user-images.githubusercontent.com/87782933/164978688-f9624e44-b7c6-4dd8-b0a4-7d9fbd306502.png)